components/comment-edit.js

```diff
@@ -37,6 +37,7 @@ export default function CommentEdit ({ comment, editThreshold, onSuccess, onCanc
           }}
           schema={commentSchema}
           onSubmit={onSubmit}
+          storageKeyPrefix={`comment-edit-${comment.id}`}
```

Comment:
is this somehow related to #2568?

Reply:
Ahh ok, that makes sense!

Sorry, I could have figured this out if I just looked at the other code. I tend to now just immediately ask if something is _really_ related if it's not immediately clear since I wasted too much time in my life trying to figure out _how_ something in someone else's code is related to a ticket or bug, just for it to end up not being related at all (and maybe even have a bug)